.kitchen.dokken.yml

```diff
@@ -75,6 +75,7 @@ platforms:
 suites:
 - name: gem
   run_list:
+    - recipe[test::dokken]
```

Comment:
You don't need this line in here do you? We're not running the Dokken recipe :)

Reply:
It creates a vagrant user, which is required. Otherwise there's no such user